web/admin/src/Admin.css

```diff
@@ -68,13 +68,13 @@ html, body, input, button, select, form{
   display: flex;
   position: relative;
 }
-@media screen and (min-width: 1230px) {
+/* @media screen and (min-width: 1230px) {
```

Comment:
Why remove this?

Reply:
Will would prefer the presentation view is always at the bottom. So this removes the shift from row to column.

Reply:
👍